routes/userapp.js

```diff
@@ -22,7 +22,9 @@ OR
 router.post('/login', function(req, res, next) {
   // Check if auth_token is provided
   if (req.body.hasOwnProperty(config.AUTH_TOKEN)) {
-    getProfileFromGoogle(req.body[config.AUTH_TOKEN], (googleName, email) => {
+    getProfileFromGoogle(req.body[config.AUTH_TOKEN],
+      (err, googleName, email) => {
```

Comment:
use named params 
http://stackoverflow.com/questions/11796093/named-parameters-in-javascript